src/ui/index.tsx

```diff
@@ -70,7 +70,11 @@ async function initializeUI({
   try {
     await registerServiceWorker();
     initializeChannels();
-    await maybeOpenOnboarding();
+
+    if (process.env.NODE_ENV !== 'test') {
```

Comment:
I'm not sure that we need such checks in the code. Or this will create a different behaviour between prod build and test run. May be it is better to provide a predefined config for test like we have some wallets in it

Reply:
Actually I noticed that it doesn't affect that single test, so I just removed this condition